installer/module/streamlit.py

```
@@ -70,6 +70,67 @@ def fetch_streamlit_apps(logger: logging.Logger) -> List[dict]:
+            max_retries=max_retries,
+            timeout=timeout,
+        )
+        variables = json.loads(json_content)
+        logger.info(
+            f"Fetched {len(variables)} integration environment variables."
+        )
+        return variables
+    except urllib.error.URLError as e:
+        logger.error(
+            f"Network error while fetching integration env vars: {e}"
+        )
```

Ruff (0.15.13)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)